crypto/keyring_message.go

```diff
@@ -89,10 +91,12 @@ func asymmetricEncrypt(data []byte, publicKey *KeyRing, privateKey *KeyRing, isB
 		FileName: "",
 	}
 
+	// We want to use the first entity only for encryption, the API will already return them in the correct order
+	// The first key is meant to be the primary
```

Comment:
This seems suspect; sometimes we need to encrypt to multiple recipients, no? If the API returns multiple keys for a single recipient, it's the responsibility of the app to pass only the first one, not the responsibility of this library.